src/main/fc/rc.c

```diff
@@ -489,141 +492,176 @@ static FAST_CODE void processRcSmoothingFilter(void)
+        feedforward += feedforwardBoost;
+        // apply jitter reduction multiplier to reduce noise by attenuating when sticks move slowly
+        feedforward *= jitterAttenuator;
+        // pull feedforward back towards zero as sticks approach max if in same direction
+        // to avoid overshooting on the outwards leg of a fast roll or flip
+        if (pid->feedforwardMaxRateLimit && feedforward * setpoint > 0.0f) {
+            const float limit = (maxRcRate[axis] - fabsf(setpoint)) * pid->feedforwardMaxRateLimit;
+            feedforward = (limit > 0.0f) ? constrainf(feedforward, -limit, limit) : 0.0f;
+        }
+
+    } else {
+        // for yaw, apply jitter reduction only to the base feedforward delta element
+        // can't be applied to the 'sustained' element or jitter values will divide it down too much when sticks are still
+        feedforward *= jitterAttenuator;
+
+        // instead of adding setpoint acceleration, which is too aggressive for yaw,
+        // add a slow-fading high-pass filtered setpoint element
+        // this provides a 'sustained boost' with low noise
+        // it mimics the normal sustained yaw motor drive requirements, reducing P and I and hence reducing bounceback
+        // this doesn't add significant noise to feedforward
+        // too little yaw FF causes iTerm windup and slow bounce back when stopping a hard yaw
+        // too much causes fast bounce back when stopping a hard yaw
+
+        // calculate lowpass filter gain factor from user specified time constant
+        const float gain = pt1FilterGainFromDelay(pid->feedforwardYawHoldTime, rxInterval);
+        pt1FilterUpdateCutoff(&feedforwardYawHoldLpf, gain);
+        const float setpointLpfYaw = pt1FilterApply(& feedforwardYawHoldLpf, setpoint);
+        // subtract lowpass from input to get highpass of setpoint for sustained yaw 'boost'
+        const float feedforwardYawHold = pid->feedforwardYawHoldGain * (setpoint - setpointLpfYaw);
+
+        DEBUG_SET(DEBUG_FEEDFORWARD, 6, lrintf(feedforward * 0.01f));  // basic yaw feedforward without hold element
+        DEBUG_SET(DEBUG_FEEDFORWARD, 7, lrintf(feedforwardYawHold * 0.01f));  // yaw feedforward hold element
+
+        feedforward += feedforwardYawHold;
+        // NB: yaw doesn't need max rate limiting since it rarely overshoots
     }
 
-    // apply feedforward transition
+    // apply feedforward transition, if configured. Archaic (better to use jitter reduction)
     const bool useTransition = (pid->feedforwardTransition != 0.0f) && (rcDeflectionAbs[axis] < pid->feedforwardTransition);
     if (useTransition) {
         feedforward *= rcDeflectionAbs[axis] * pid->feedforwardTransitionInv;
     }
 
-    // apply averaging
-    if (feedforwardAveraging) {
-        feedforward = laggedMovingAverageUpdate(&feedforwardDeltaAvg[axis].filter, feedforward);
-    }
+    if (axis == (int)gyro.gyroDebugAxis) {
```

Comment:
```suggestion
    if (axis == gyro.gyroDebugAxis) {
```

Reply:
Better solution to fix it in the header, done